
controller_manager:
  ros__parameters:
    update_rate: 100  # Hz

    # Broadcaster que saca JointStates reales
    # joint_state_broadcaster:
    #   type: joint_state_broadcaster/JointStateBroadcaster

    effort_controller:
      type: effort_controllers/JointGroupEffortController

    steering_controller:
      type: position_controllers/JointGroupPositionController

# joint_state_broadcaster:
#   ros__parameters:
#     extra_joints: '{}'
#     interfaces: '{}'
#     joints: '{}'
#     map_interface_to_joint_state:
#       effort: effort
#       position: position
#       velocity: velocity
#     use_local_topics: false   

effort_controller:
  ros__parameters:
    joints:
      - ED_IZQ_joint
      - ED_DCH_joint
      - ET_IZQ_joint
      - ET_DCH_joint

steering_controller:
  ros__parameters:
    joints:
      - MGD_DCH_joint
      - MGD_IZQ_joint